conan/tools/gnu/pkgconfigdeps.py

```diff
@@ -226,13 +228,8 @@ class PkgConfigConan(ConanFile):
 
             def package_info(self):
                 self.cpp_info.requires = ["other::cmp1"]
-        ```
-        Or:
 
-        ```python
-        from conan import ConanFile
-        class PkgConfigConan(ConanFile):
-            requires = "other/1.0"
+            # Or:
```

Comment:
Why has it been removed?

Reply:
Oh, ok, I saw it 😅  I preferred to duplicate the example instead of this, but I think it's OK as well

Reply:
Not removed, just compacted, reusing the same part above, and putting the OR inside the ``package_info()``, so it is easier to see what is the difference.